container-disc/src/main/sh/vespa-start-container-daemon.sh

```diff
@@ -71,6 +71,11 @@ configure_memory() {
             available="$VESPA_TOTAL_MEMORY_MB"
         else
             available=`free -m | grep Mem | tr -s ' ' | cut -f2 -d' '`
+            if [ -x "$(command -v cgget)" ]; then
```

Comment:
This doesn't work well unless cgget exists and is a program:

```
$ command -v foobar
$ command -v ls
alias ls='ls -G'
```

Looks like `hash` is ideal for this case:

```
if hash &>/dev/null; then
```